fix: Add backwards compatability with canvas provider (#3773)
Workday/canvas-kit: modules/docs/llm/upgrade-guides/15.0-UPGRADE-GUIDE.md

`Menu`, `Modal`, `Popup`, `Toast` and `Tooltip`

## Theming

### System Brand Tokens and Brand Tokens

> The relationship between **system brand tokens** (e.g. `system.color.brand.accent.primary`) and **brand tokens** (e.g. `brand.primary600`) has changed. Teams can still set palette values such as `base`, `dark`, `darkest`, `light`, `lightest`, and `lighter` via the `CanvasProvider` theme prop. The mapping inside `CanvasProvider` exists for **backwards compatibility**: when you pass a theme object, we forward those values to both the legacy brand tokens and the system brand tokens so existing usage continues to work.

Reviewer: ```suggestion
The relationship between **system brand tokens** (e.g. `system.color.brand.accent.primary`) and **brand tokens** (e.g. `brand.primary600`) has changed. Teams can still set palette values such as `base`, `light`, `lighter`, lightest`, `dark`, `darker` and `darkest` via the `CanvasProvider` theme prop. The mapping inside `CanvasProvider` exists for **backwards compatibility**. When you pass a theme object, we forward those values to both the legacy brand tokens and the system brand tokens so current implementations will continue to work.
```

For more information on theming, view our [Theming](https://workday.github.io/canvas-kit/?path=/docs/features-theming-overview--docs) documentation.

> For more information on our tokens, view our [Tokens](https://workday.github.io/canvas-tokens/?path=/docs/docs-getting-started--docs) documentation.

Reviewer: This is to just add a line break for these links from the paragraph above.

```suggestion

For more information on theming, view our [Theming](https://workday.github.io/canvas-kit/?path=/docs/features-theming-overview--docs) documentation.

For more information on our tokens, view our [Tokens](https://workday.github.io/canvas-tokens/?path=/docs/docs-getting-started--docs) documentation.
```

> 

Reviewer: ```suggestion

```tsx
<CanvasProvider theme={{canvas: customColorTheme}}>
      ....
</CanvasProvider>
```
```
